api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -357,7 +330,6 @@ public DbUser createUser(
       DbVerifiedInstitutionalAffiliation dbVerifiedAffiliation) {
     DbUser dbUser = new DbUser();
     dbUser.setCreationNonce(Math.abs(random.nextLong()));
-    dbUser.setDataAccessLevelEnum(DataAccessLevel.UNREGISTERED);
```

Comment:
Users with no DbUserAccessTier entries are UNREGISTERED.  So this is no longer necessary.